Flowchart/obj/Debug/netcoreapp1.0/FlowchartCreator.csproj.FileListAbsolute.txt

```diff
@@ -8,3 +8,23 @@ C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\ne
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\FlowchartCreator.pdb
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\UserSecretsAssemblyInfo.cs
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\FlowchartCreator.csprojResolveAssemblyReference.cache
+C:\Users\Sam\Desktop\Flowchart-Creator-production\Flowchart\bin\Debug\netcoreapp1.0\FlowchartCreator.deps.json
```

Comment:
Yeah I think so, but I didn't want to mess anything up for anyone else. Is there anything in the netcoreapp1.0 folder that we would need? Ca I just ignore the whole thing?